packages/har-examples/src/index.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable camelcase */
+import type { Har } from 'har-format';
+
+import application_form_encoded from './application-form-encoded.har.js';
+import application_json from './application-json.har.js';
+import application_zip from './application-zip.har.js';
+import cookies from './cookies.har.js';
+import full from './full.har.js';
+import headers from './headers.har.js';
+import https from './https.har.js';
+import image_png_no_filename from './image-png-no-filename.har.js';
+import image_png from './image-png.har.js';
+import jsonObj_multiline from './jsonObj-multiline.har.js';
+import jsonObj_null_value from './jsonObj-null-value.har.js';
+import multipart_data_dataurl from './multipart-data-dataurl.har.js';
+import multipart_data from './multipart-data.har.js';
+import multipart_file from './multipart-file.har.js';
+import multipart_form_data from './multipart-form-data.har.js';
+import query_encoded from './query-encoded.har.js';
+import query from './query.har.js';
+import short from './short.har.js';
+import text_plain from './text-plain.har.js';
+import xml from './xml.har.js';
+
+const examples = {
+  'application-form-encoded': application_form_encoded satisfies Har as Har,
```

Comment:
it's still rather ugly but i think this is a step in the right direction and i'm actually quite glad TS suggested this:

![CleanShot 2025-03-06 at 12 00 13@2x](https://github.com/user-attachments/assets/84bb9f03-0fd1-4520-a7c7-5fad7f8d87ea)

with `satisfies T as T` we can at least assert that the object in question meets the requirements of `T` before setting it with `as`